src/uct/ib/mlx5/ib_mlx5.h

```diff
@@ -18,6 +19,9 @@
 #include <infiniband/arch.h>
 #include <netinet/in.h>
 #include <string.h>
+#if __ARM_NEON
```

Comment:
and then include ucs/arch/arm here ? Or it is already inherited through one of the above headers ?